sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,2 @@
+SELECT * FROM customers
+WHERE "isActive" = 'false';
```

Comment:
Does this one work? You wouldn't usually quote a boolean. It probably works because Postgres does aggressive data conversion.

Reply:
I initially had it without the quotation marks, but thought it didn't work. It turns out there were not any records where isActive = false. I have changed it back to its original state without quotation marks.